src/MessageBird/Resources/AvailablePhoneNumbers.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace MessageBird\Resources;
+
+use MessageBird\Objects;
+use MessageBird\Common;
+
+/**
+ * Class AvailablePhoneNumbers
+ *
+ * @package MessageBird\Resources
+ */
+class AvailablePhoneNumbers
+{
+
+    /**
+     * @var \MessageBird\Common\HttpClient
+     */
+    protected $HttpClient;
+
+    /**
+     * @param Common\HttpClient $HttpClient
+     */
+    public function __construct(Common\HttpClient $HttpClient)
+    {
+        $this->HttpClient = $HttpClient;
+    }
+
+    /**
+     * @param string $countryCode
+     * @param array $parameters
+     *
+     * @return Objects\BaseList
+     * @throws \MessageBird\Exceptions\RequestException
+     * @throws \MessageBird\Exceptions\ServerException
+     */
+    public function getList($countryCode, $parameters = array())
+    {
+        list($status, , $body) = $this->HttpClient->performHttpRequest(
+            Common\HttpClient::REQUEST_GET,
+            "available-phone-numbers/$countryCode",
+            $parameters
+        );
+
+        if ($status === 200) {
```

Comment:
Why not the other way around? Then it will be easier to read. Just a suggestion

```suggestion
        if ($status !== 200) {
             return $this->processRequest($body);
        }
```